docs/cunumeric/source/user/notebooks/stencil.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "14c01af2-d331-4aa4-9639-41353601a767",
+   "metadata": {},
+   "source": [
+    "# Jacobi stencil\n",
+    "\n",
+    "<details>\n",
+    "<summary>License</summary>\n",
+    "<pre>\n",
+    "\n",
+    "Copyright 2023 NVIDIA Corporation\n",
+    "\n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "you may not use this file except in compliance with the License.\n",
+    "You may obtain a copy of the License at\n",
+    "\n",
+    "     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "\n",
+    "Unless required by applicable law or agreed to in writing, software\n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "See the License for the specific language governing permissions and\n",
+    "limitations under the License.\n",
+    "</pre>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35c48e6f-1bde-4aac-af55-b7218cc22491",
+   "metadata": {},
+   "source": [
+    "To get started, `import cunumeric as np` (just the same way we would import `numpy`)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "bfc50224-f762-4ec0-8ded-671cbd1df559",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import cunumeric as np  # instead of numpy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "19f09f10-a627-4ca2-bf7c-a75c71f54ac5",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from legate.timing import time\n",
```

Comment:
timing should probably be explained as in the other notebooks. We may actually want to mention about the asynchronous nature of the computation and why regular python timing will not work. Actually, that might be a great 4th notebook.  Show python time giving 0ms because everything is deferred and legate timing giving you real times.

Reply:
A notebook for timing seems good, do we also want to have a sentence or two in all the notebooks for reinforcement? If so, can you suggest some verbiage?

Reply:
Maybe...


Because of Legate's deferred execution model, `legate.timing` should be used instead of standard Python `datetime` utilities. Python `datetime.now` would return the time a task is *enqueued*, not necessarily the time a task finishes executing.

@manopapad @magnatelee

Reply:
https://github.com/nv-legate/cunumeric/pull/805/commits/940fe8ad19b139fab627f6db0c80df61d7a90eeb